cmd/sriov/main.go

```diff
@@ -74,7 +74,24 @@ func cmdAdd(args *skel.CmdArgs) error {
 	defer netns.Close()
 
 	sm := sriov.NewSriovManager()
-	if err := sm.ApplyVFConfig(netConf); err != nil {
+	pfLink, err := sm.FillOriginalVfInfo(netConf)
+	if err != nil {
+		return fmt.Errorf("failed to lookup master %q: %v", netConf.Master, err)
```

Comment:
error string should be: "failed to get original vf information: %v"

the needed info is already part of `err`